include/sound/sof/topology.h

```diff
@@ -33,6 +33,8 @@ enum sof_comp_type {
 	SOF_COMP_EQ_IIR,
 	SOF_COMP_EQ_FIR,
 	SOF_COMP_KEYWORD_DETECT,
+	SOF_COMP_KPB, /* A key phrase buffer component */
+	SOF_COMP_SELECTOR,	/**< channel selector component */
```

Comment:
SOF_COMP_CH_SELECTOR?

Reply:
@plbossart this will have to change in the FW as well. Let me submit a PR for the FW too